plio/models.py

```diff
@@ -158,3 +158,17 @@ def video_update_cache(sender, instance, **kwargs):
     plios = Plio.objects.filter(video_id=instance.id)
     # invalidate saved cache for the plios
     invalidate_cache_for_instances(plios)
+
+
+@receiver(post_save, sender=Item)
+@receiver(post_delete, sender=Item)
+def item_update_cache(sender, instance, **kwargs):
+    # invalidate saved cache for the plio
+    invalidate_cache_for_instance(instance.plio)
```

Comment:
this should also invalidate the cache for question linked to that item too right?

Reply:
so we're not caching question also. Only the whole plio response. So we are resetting the whole plio cache and not individual question.